ROOTFS/etc/waggle/sanity/fatal/rpi-present.test

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+echo "RPi Present Test: Begin"
+
+expectedRPi=$(cat /etc/waggle/node_manifest.json | jq '[.shield | select(.present == true)] | length')
+
+if [[ $expectedRPi == 0 ]]; then
+    echo "RPi Present Test: No shield expected for this device PASS"
+    exit 0
+fi
+
+nmapOut=$(nmap -sP 10.31.81.1/24)
```

Comment:
The rpi is set to come up as 10.31.81.4 via our dnsmasq rules.  maybe we additionally want to test for specifically that?